### PR TITLE
Check PR branch names

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
Checks that no PR is created from a branch named *develop* or from a branch name that starts with *release-* or *test-*

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by jameesjohn** [comment]
This work has been tested here https://github.com/jameesjohn/comment-on-pr/pull/12

**Comment by jameesjohn** [comment]
PTAL @kevinlee12 @ankita240796

File: lib/checkPullRequestBranch.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        in another branch and send in a PR.`
+    });
+    const updatePRParams = context.issue({
+      state: 'closed'
+    });
+    // eslint-disable-next-line no-console
+    console.log('Updating pull request');
+
+    await context.github.issues.createComment(stateCommentParams);
+    await context.github.issues.edit(updatePRParams);
```
Comment (jameesjohn) [inline lib/checkPullRequestBranch.js:20]: In the new API, this will be context.github.issues.update()

**Comment by kevinlee12** [comment]
hmm, why is this in this version of Probot? I thought all new functionality was going to be in a **new** version of Probot?

**Comment by jameesjohn** [comment]
Yes, it is supposed to be in a new version, but I was supposed to create a PR for GSoC and I didn't so I had to create one. 
During the upgrade, the change will just be in the one place specified here

**Comment by kevinlee12** [comment]
Hmm, okay. Even if the code looks okay, I’m going to strongly discourage that this PR gets merged. The new Probot uses typescript and has a completely different API.

**Comment by jameesjohn** [comment]
Hmm, okay.

**Comment by ankita240796** [comment]
Hey @jameesjohn, Thanks for making the PR but I agree with @kevinlee12 that we should not merge in more changes to this outdated version. So, here is what I would suggest, focus more on writing down how you are going to upgrade to the latest version and you can skip the contribution part for now. Just link in the PR as one of the contributions but do not proceed further in this direction. Thanks!

**Pull request closed without merging**